### DISCUSSION TITLE
Parse custom date move from moment to luxon

### DISCUSSION THREAD

**Discussion by axi92** [moment/luxon#1407] [Q&A] [answered]
I used moment in this project but I wanted to move to luxon.
My old moment code:
https://github.com/axi92/steam-workshop-scraper/blob/master/index.js#L157

I tried this:
But there was no format to parse "3:40pm"
`let aaa = DateTime.fromFormat(string, "d MMM, yyyy @ hh:mma", { locale: "en" });`

I am parsing 2 types of formats from a website:
First: `17 Mar @ 2:17pm`
Second: `13 Sep, 2015 @ 6:52pm`

**Comment by diesieben07** [comment, marked as answer]
`d MMM, yyyy @ hh:mma` cannot parse `13 Sep, 2015 @ 6:52pm`, because `hh` and `mm` are _padded_ tokens. That means the format expects _exactly_ two digits. Your input strings use unpadded numbers, so you should use `h` and `m`.

`d MMM @ h:ma` can be used to parse your 1st example.
`d MMM, yyyy @ h:ma` can be used to parse your 2nd example.

Please let me know if this solves your issue!

**Reply by axi92** [reply]
Thank you that helped a lot =)
